rustfmt.toml

```diff
@@ -0,0 +1,5 @@
+format_strings = true
```

Comment:
I suggest to remove these unrelated changes from the PR and revert all formatting changes.